.travis.yml

```diff
@@ -1,4 +1,4 @@
-sudo: required
+sudo: false
```

Comment:
@tarungarg546 I put to perform a test and it was my fault not to remove it.. You can remove this 👍